ntbs-service/DataAccess/NtbsContext.cs

```diff
@@ -468,6 +468,8 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                         i.Property(e => e.NotifiedToPheStatus)
                             .HasConversion(statusEnumConverter)
                             .HasMaxLength(EnumMaxLength);
+                        i.Property(e => e.RelationshipToCase)
+                            .HasMaxLength(90);
```

Comment:
Do we actually need this, given that we've specified it in the attribute?

Reply:
You're right, we don't. Have removed now.